src/Cli/dotnet/commands/dotnet-pack/PackCommandParser.cs

```diff
@@ -14,6 +14,8 @@ namespace Microsoft.DotNet.Cli
     internal static class PackCommandParser
     {
         public static readonly string DocsLink = "https://aka.ms/dotnet-pack";
+        public static readonly string CustomDefaultConfigurationProperty = "PackRelease";
+        public static string enablePackReleaseSolutionEnvironmentVar = "DOTNET_CLI_ENABLE_PACK_RELEASE_FOR_SOLUTIONS";
```

Comment:
Are we tracking documentation changes for the new variables?

Reply:
I had a PR for doc changes before these worked in `.csproj` and `sln`s but haven't made one for these changes. I'll do that now and link it here when I'm done, thank ya!

Reply:
Added some docs here :) https://github.com/dotnet/docs/pull/30747